src/cljc/harja/domain/kanavat/kanavan_huoltokohde.cljc

```diff
@@ -13,4 +13,6 @@
 
 (def perustiedot
   #{::id
-    ::nimi})
\ No newline at end of file
+    ::nimi})
+
+(s/def ::hae-huoltokohteet-kysely ::huoltokohde)
```

Comment:
Vai `coll-of ::huoltokohde`?

Reply:
Joo näimpä. Omituista että tuo yksittäinen muoto on myös toiminut.